database/mysql.go

```diff
@@ -94,30 +99,57 @@ func (my *mysql) Connect(ctx context.Context) (*sql.DB, error) {
 	return db, nil
 }
 
-func mysqlConnection(logger log.Logger, user, pass string, address string, database string, sslCA string) *mysql {
+func mysqlConnection(logger log.Logger, user, pass string, address string, database string, tlsCAFile string) (*mysql, error) {
 	timeout := "30s"
 	if v := os.Getenv("MYSQL_TIMEOUT"); v != "" {
 		timeout = v
 	}
+	params := fmt.Sprintf(`timeout=%s&charset=utf8mb4&parseTime=true&sql_mode="ALLOW_INVALID_DATES,STRICT_ALL_TABLES"`, timeout)
 
-	var params []string
-	params = append(params, fmt.Sprintf("timeout=%s", timeout))
-	params = append(params, "charset=utf8mb4")
-	params = append(params, "parseTime=true")
-	params = append(params, `sql_mode="ALLOW_INVALID_DATES,STRICT_ALL_TABLES"`)
+	var tlsConfig *tls.Config = nil
 
-	if sslCA != "" {
-		params = append(params, fmt.Sprintf("sslca=%s", sslCA))
+	if tlsCAFile != "" {
+		tlsConfig = &tls.Config{
+			InsecureSkipVerify: true,
```

Comment:
This seems like something we should accept on the `DatabaseConfig` rather than hardcode deep in our library.

Reply:
I figured that since we were doing the CA check this covered that. I'll create a config option for this and for our CA-only check so that configurations can choose which combination to enforce.